docs/requirements/requirements.md

```diff
@@ -0,0 +1,310 @@
+- It should be really easy and intuitive to quickly deploy the system.
+- The RA should not depend on other systems or services.
+- Origin: Researchers
+
+#### R-U-006: Transparency
+
+- Researchers must be able to judge quickly whether or not the RA is of good use for them: The documentation or public appearance of the RA must be appealing, comprehensible, complete and informative so that interested parties can easily assess whether the RA meets their requirements.
+- Researchers must have access to the code and the documentation.
+- The code must be open source.
+- One researchers said that she does not like that one simply does not know exactly how the existing RAs compare to each other since their documentations lacking in quality and quantity.
+- Origin: Supervisors, Researchers
+
+#### R-U-007: The domain must be self-explanatory
+
+- The domain must not be an obstacle under any circumstances. As long as the domain is *self-explanatory*, it doesn't matter which domain is shown.
+- The domain must not require special knowledge about it in order to be able to use the system.
+- Origin: Researchers
+
+#### R-U-008: Observability
+
+- The user must be able to observe and understand exactly what the system is doing or last did.
+- The RA must not try to be a blackbox.
+- Origin: Examiner, Supervisors, Researchers
+
+#### R-U-009: Visualization of load
+
+- The current utilization (load) of the individual microservices should be visualized at runtime.
+- The system should show the current differences in load for the individual services (e.g., using thick lines for more requests).
+- Origin: Examiner
+
+#### R-U-010: Explanations of metrics
+
+- Metrics that could be interesting to researchers (e.g. regarding performance) should be well explained.
+- Origin: Researcher
+
+#### R-U-011: Adding services
+
+- It should be easy to add more microservices to the system.
+- Origin: Researcher
+
+#### R-U-012: True to the term *Reference Architecture*
+
+- The system should be usable as a reference system in the context of education. One should be able to show, let observe, and explain what a microservice architecture is and how it reacts to load.
+- Also one should be able to observe and better understand the effects of scaling.
+- Origin: Researcher
+
+#### R-U-013: Deterioration in performance
+
+- It should be fairly easy to control the performance of the system. A researcher should be able to decide when and how the performance deteriorates.
+- Origin: Researcher
+
+#### R-U-014: Demo mode
+
+- There should be a kind of demo mode so that a new user can try out the system quickly and easily: We provide test data with which the system can be directly *loaded*.
+- Origin: Developers
+
+### Non-functional Requirements
+
+#### R-NFR-001: Performance
+
+- The microservices must be performant.
+- Origin: Examiner
+
+#### R-NFR-002: Elasticity
+
+- The system must be *elastic*.
```

Comment:
maybe some more elaboration

Reply:
Yes, that would be nice, I added this as a to-do / open question.